modules/apps/object/object-web/src/main/java/com/liferay/object/web/internal/object/definitions/display/context/ObjectDefinitionsViewsDisplayContext.java

```diff
@@ -69,6 +75,91 @@ public List<FDSActionDropdownItem> getFDSActionDropdownItems()
 				"delete", "delete", "async"));
 	}
 
+	public JSONArray getWorkflowStatusJSONArray() {
+		return JSONUtil.putAll(
+			JSONUtil.put(
+				"label",
+				LanguageUtil.get(
+					_objectRequestHelper.getRequest(),
+					WorkflowConstants.LABEL_APPROVED)
+			).put(
+				"value", WorkflowConstants.LABEL_APPROVED
```

Comment:
Hi @danvb112, the value must be WorkflowConstants.STATUS_APPROVED because the status is persisted as a number in the data base,

Could you replace the other values as well?

Thanks!